plugins/proxy-backend/README.md

```diff
@@ -14,13 +14,13 @@ To run it within the backend do:
 
 1. Register the router in `packages/backend/src/index.ts`:
 
-```
+```tsx
```

Comment:
just `ts` is fine here.

Reply:
It's used elsewhere in the docs, should we change all of them? @benjdlambert

Reply:
I mean it really doesn't matter to be honest, it's just syntax highlighting. .tsx is when you normally have some React thrown in the mix, which isn't gonna happen in backend stuff. And I'm pretty sure that the colours are gonna be the same either way hah

Reply:
Good point. Fixing.